gh001-workflows/caller-workflows/build-push-dev.yaml

```diff
@@ -0,0 +1,28 @@
+name: build-push-dev
+
+on:
+  push:
+    tags:
+      - '[0-9].[0-9].[0-9]rc*'
```

Comment:
This can be reduced.

Reply:
I have tried a couple of regex combinations, but I haven't gotten one that is working